capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -32,7 +34,7 @@
    * @param numOfWomen A float that represents the number of women enrolled at the undergradute
    *     level.
    */
-  public static ScorecardData create(
+  private static ScorecardData create(
```

Comment:
Also - should have caught this earlier, do we actually need the `create` method yet? I don't think GSON is going to use this. We will want to create instance of this class by hand when we test translating it into our domain object - but I think will be more readable with [builders](https://github.com/google/auto/blob/master/value/userguide/builders.md) - and we can add them when we get there.

In summary let's:
* Remove the create method entirely
* Migrate the @param javadocs to the individual accessor methods 
* Add that note about the URL where the descriptions came from in the Javadoc for the class

Reply:
PS - this is why having the tests in the same PR is sometimes helpful. We could verify that removing the create method is ok (I'm pretty sure it is 😃 ).

Reply:
the test has been moved into this PR :)